Polish and cleanup (#1)
neoforged/Documentation: neogradle/docs/index.md

---
sidebar_position: 0
---

NeoGradle Documentation
=========================

:::caution
Please note that this documentation may not be up to date considering the recent creation of NeoForged.

> Until NeoGradle releases it's first version, you should refer to ForgeGradle documentation. The ForgeGradle documentation for versions 6 and 5 have been archived here.

Reviewer: ```suggestion
Until NeoGradle releases its first version, you should refer to ForgeGradle documentation. The ForgeGradle documentation for versions 6 and 5 has been archived here.
```
